website/src/pages/case-studies/nutanix-fixed-log4j-with-sourcegraph.tsx

```diff
@@ -0,0 +1,332 @@
+                <blockquote>
+                    <div className="my-4">
+                        <img src="/external-logos/nutanix-logo.svg" width="210px" alt="nutanix" />
+                    </div>
+                    <h2 className="display-3 fw-550 mw-600 mx-auto">
+                        "Sourcegraph was the right product at the right time."
+                    </h2>
+                    <footer className="blockquote-footer">
+                        Jon Kohler, Technical Director of Solution Engineering at Nutanix
+                    </footer>
+                </blockquote>
+            </ContentSection>
+
+            <section className="d-flex flex-column flex-md-row">
+                <div className="bg-light-gray-2 p-lg-6 p-md-5 px-1 py-5 col-sm-12 col-md-6">
+                    <div className="mb-5 ml-md-5 ml-3 d-flex flex-column flex-md-row">
+                        <div className="d-flex bg-white align-self-center align-self-md-start col-1 col-md-2 justify-content-center align-items-center p-0 rounded-circle text-center pt-1 mw-50">
+                            <ClipboardTextOutlineIcon color="#00A1C7" size={40} className="p-1" />
+                        </div>
+                        <div className="pl-3 col-11 col-lg-9">
+                            <h5 className="fw-550">Use case</h5>
+                            <p>Find and fix security vulnerabilities quickly across the code base.</p>
+                        </div>
+                    </div>
+                    <div className="mb-5 ml-md-5 ml-3 d-flex flex-column flex-md-row">
+                        <div className="d-flex bg-white align-self-center align-self-md-start col-1 col-md-2 justify-content-center align-items-center p-0 rounded-circle text-center pt-1 mw-50">
+                            <FlagOutlineIcon color="#00A1C7" size={40} className="p-1" />
+                        </div>
+                        <div className="pl-3 col-11 col-lg-9">
+                            <h5 className="fw-550">Challenge</h5>
+                            <ul className="pl-4">
+                                <li>
+                                    Inability to efficiently find all the instances of Log4j in their large codebase.
+                                </li>
+                                <li>Lack of code-level clarity that vulnerabilities were completely resolved.</li>
+                                <li>Uncertainty about the scope and impact of the Log4j vulnerability.</li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div className="mb-5 ml-md-5 ml-3 d-flex flex-column flex-md-row">
+                        <div className="d-flex bg-white align-self-center align-self-md-start col-1 col-md-2 justify-content-center align-items-center p-0 rounded-circle text-center pt-1 mw-50">
+                            <CheckCircleOutlineIcon color="#00A1C7" size={40} className="p-1" />
```

Comment:
The borders on some of these look uneven to me?
![Screen Shot 2022-03-31 at 11 42 17 AM](https://user-images.githubusercontent.com/43454804/161095494-92e05c43-a021-4077-b7e1-5884e23538f6.png)
?

Reply:
Culprit was a stray `pt-1` 😅 All good now